### PR TITLE
feat(tests): add integration tests

### PR DESCRIPTION
**Pull request by FrodoTheTrue**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      with:
+        instance_name: '${{ secrets.SSH_COMPUTE_IAP_VM_NAME }}'
+        zone: 'us-central1-a'
+        ssh_private_key: '${{ secrets.GCP_SSH_PRIVATE_KEY }}'
+        command: 'uname -r'
+
+    - name: 'outputs'
+      run: echo '${{ steps.ssh-compute.outputs.stdout }}${{ steps.ssh-compute.outputs.stderr }}'
+
+  auth_credentials:
```
Comment (sethvargo) [inline .github/workflows/integration.yml:51]: I didn't catch it in the initial review (my fault), but I don't think we should allow direct credentials in new actions - we should push everything through `auth` instead.

Comment (FrodoTheTrue) [reply .github/workflows/integration.yml:51]: Hm, it's an example of using json auth through auth action. Should we don't use json auth at all?

Comment (sethvargo) [reply .github/workflows/integration.yml:51]: Oh derp. I mis-read. This is fine.

**Review by sethvargo — APPROVED** [review]

**Comment by FrodoTheTrue** [comment]
PR to origin: https://github.com/google-github-actions/ssh-compute/pull/7

**Pull request closed without merging**